webapp-django/crashstats/crashstats/views.py

```diff
@@ -723,10 +723,6 @@ def crashes_per_day(request, default_context=None):
     for version in context['active_versions'][params['product']]:
         context['available_versions'].append(version['version'])
 
-    if not params.get('platforms'):
-        params['platforms'] = [
-            x['name'] for x in platforms if x.get('display')
-        ]
```

Comment:
If you go to `https://crash-stats.mozilla.com/crashes-per-day/?p=Firefox` it used to automatically added `{'platform': ['Windows', 'Linux', 'Mac OS X']}` to the parameters sent to SuperSearch. But if you go to https://crash-stats.mozilla.com/crashes-per-day/?p=FennecAndroid it would do that same thing and because the platform for all FennecAndroid crashes is "Android" that extra supersearch parameter would excluded all crashes. 

Secondly, the UX *used* to be that all checkboxes are checked by default so once you, for example, change the dates on that page and click the "Generate" button it would add `&platforms=Windows&platforms=Linux&platforms=Mac%20OS%20X` to the query string. 

What I'm changing is basically that if nothing is selected (or all three options are selected) the whole `platform` parameter is totally ignored in terms of fetching the data.